distribution/docs/src/main/resources/_contents/_tables/org.codice.alliance.mgmp.source.adoc

```diff
@@ -0,0 +1,131 @@
+.[[MGMP_Federated_Source]]GMD Federated Source
```

Comment:
Do you think it makes sense to have this since there isn't an MGMP specific federated source and the Gmd Source is already documented on the DDF side?